module/documents/actor/actor.mjs

```diff
@@ -1415,6 +1415,11 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
     const relevant = type === "skill" ? this.system.skills?.[config.skill] : this.system.tools?.[config.tool];
     const buildConfig = this._buildSkillToolConfig.bind(this, type, config.rolls?.shift());
 
+    if ( config.ability === "spellcasting" ) {
```

Comment:
Unless there is a particular reason to be able to use this in the rolling methods, I'd probably prefer to handle converting "spellcasting" to a specific ability in the `rollAction` method. Would allow us to write the logic only once. Not going to fight about it though.